airbyte-webapp/src/views/Connector/ConnectorDocumentationLayout/ConnectorDocumentationLayout.tsx

```diff
@@ -14,9 +14,10 @@ const LazyDocumentationPanel = lazy(() =>
 
 export const ConnectorDocumentationLayout: React.FC<React.PropsWithChildren<unknown>> = ({ children }) => {
   const { formatMessage } = useIntl();
-  const { documentationPanelOpen } = useDocumentationPanelContext();
+  const { documentationPanelOpen, documentationUrl } = useDocumentationPanelContext();
   const screenWidth = useWindowSize().width;
-  const showDocumentationPanel = screenWidth > 500 && documentationPanelOpen;
+  const showDocumentationPanel =
+    screenWidth > 500 && documentationPanelOpen && documentationUrl.includes("docs.airbyte.com");
```

Comment:
These documentation URLs come from the source/destination_definition seed yaml files, so the internal docs _always_ include `docs.airbyte.com` at this level.  (the panel itself does the stripping of the URL and pulling from the build)

Reply:
Seems like a fine way to detect whether to show the panel! Just a minor suggestion to hint at why we're looking for this substring, take it or leave it:

```suggestion
    const isOfficialDocumentation = documentationUrl.includes("docs.airbyte.com");
    screenWidth > 500 && documentationPanelOpen && isOfficialDocumentation;
```